lib/components/channel_panel.dart

```diff
@@ -157,8 +188,44 @@ class ChannelPanelWidget extends StatelessWidget {
               ),
             ]),
           );
-        })
+        }),
+        _buildEmotePicker(context)
       ]);
     });
   }
+
+  Offstage _buildEmotePicker(BuildContext context) {
```

Comment:
This might be better off as a separate widget. Feel free to do that in a separate PR though.